draft-ietf-anima-brski-prm.md

```diff
@@ -319,12 +332,12 @@ Based on the intended target environment described in {{sup-env}}, the following
 * By contrast, the Registrar-Agent can be authenticated by the registrar as a component, acting on behalf of the registrar.
   In addition the registrar must be able to verify, which Registrar-Agent was in direct contact with the pledge.
 
-* It would be inaccurate for the voucher-request and voucher-response to use an assertion with value "proximity" in the voucher, as the pledge was not in direct contact with the registrar for bootstrapping.
+* It would be inaccurate for the voucher-request and voucher-response to use an assertion with value `proximity` in the voucher, as the pledge was not in direct contact with the registrar for bootstrapping.
   Therefore, a new Agent-Proximity Assertion value {#agt_prx} is necessary for distinguishing assertions the MASA can state.
 
```

Comment:
should this be {{agt_prx}} here?

Reply:
This is an issue already existing in Draft 12 -- I actually skipped the requirements section in this iteration, so good catch!

Since the requirements should not talk about solution space, I now changed it to:

> Therefore, a new assertion type is necessary for distinguishing assertions the MASA can state

Reply:
okay, true, that is just the overview.